src/rules/tsUtils.ts

```diff
@@ -96,6 +96,19 @@ export enum HookName {
   'afterEach' = 'afterEach',
 }
 
+export enum DescribeProperty {
```

Comment:
no need to export them, but meh

Reply:
Yeah, I've just tried to be consistent with all of the rest, but you are right, it is not necessary at the moment.